docs/getting_started/important_concepts.md

```diff
@@ -0,0 +1,110 @@
+---
+icon: material/lightbulb
+---
+
+# **Important Concepts**
```

Comment:
Could be especially useful to have a high level diagram to anchor all the dense information described below, a "bird's eye" view of OG being connected with the broader BEHAVIOR ecosystem